### PR TITLE
Fix 404.

### PR DESCRIPTION
**Pull request by veelo**

### CODE REVIEW

File: std/experimental/allocator/building_blocks/kernighan_ritchie.d [hunk 1]
```diff
@@ -8,7 +8,7 @@ version(unittest) import std.conv : text;
 // KRRegion
 /**
 $(D KRRegion) draws inspiration from the $(LINK2
-std_experimental_allocator_region.html, region allocation strategy) and also the
+std_experimental_allocator_building_blocks_region.html, region allocation strategy) and also the
```
Comment (aG0aep6G) [inline std/experimental/allocator/building_blocks/kernighan_ritchie.d]: `LINK2` shouldn't be used for phobos references. It doesn't work on the /library/ pages.

Use `MREF` instead: `$(MREF std,experimental,allocator,building,blocks,region)`

Comment (veelo) [inline std/experimental/allocator/building_blocks/kernighan_ritchie.d]: Should that be $(MREF std,experimental,allocator,building,blocks,region, region allocation strategy) instead, or where does the link text go?

Comment (aG0aep6G) [inline std/experimental/allocator/building_blocks/kernighan_ritchie.d]: Oh, didn't realize that it has custom text. Use `MREF_ALTTEXT` then; text goes first:

    $(MREF_ALTTEXT region allocation strategy, std,experimental,allocator,building_blocks,region)

(I had also made a mistake with `building_blocks`. The underscore is part of the name there.)

**Comment by veelo** [comment]
Is it OK to break lines inside macro's like this?

**Comment by CyberShadow** [comment]
Auto-merge toggled on

**Pull request merged**